invenio_cli/commands/local.py

```diff
@@ -108,7 +109,9 @@ def update_statics_and_assets(self, force, debug=False, log_file=None):
                     if op[-1] in messages:
                         click.secho(messages[op[-1]], fg="green")
                     response = run_interactive(
-                        op, env={"PIPENV_VERBOSITY": "-1"}, log_file=log_file
+                        op,
+                        env={"PIPENV_VERBOSITY": "-1", **js_pkg_man.env_overrides()},
```

Comment:
here we tell `invenio` about the JS package manager that we want to use, via environment variables